src/frameworks.sources

```diff
@@ -2256,6 +2257,7 @@ MACCATALYST_FRAMEWORKS = \
 	JavaScriptCore \
 	LinkPresentation \
 	LocalAuthentication \
+	MailKit \
```

Comment:
why ? it seems types are all decorated with `[NoMacCatalyst]` ?

Reply:
I was a little confused about this, but this makes sense not to add it, thanks!